### ISSUE TITLE
X-Axis Log option not functioning as suggested

### ISSUE THREAD

**Issue by edaphos** [automeris-io/WebPlotDigitizer#327]
After calibrating X-Axis for log scale (as described in protocol) the digitized values returned are incorrect.
![Safety Data Covid-19](https://github.com/automeris-io/WebPlotDigitizer/assets/44734299/6f460c8e-b706-4cc6-8d83-33c90b810686)
Please advise.

**Comment by nbehrnd** [comment]
Your report is thin.  Did you use off-line version 4 (if so, OS?) or the
online version 5?  What diagram, and what trace you processed / aimed to
process?  If you got some digitized data, what are the readings?

Opting for a manual pick (without KI support) of the solid blue squares
(MOD/Expected) in the fourth plot (NE_GSZ), I obtained these readings:

```
49.451, 3.9666
359.56, 2.9817
1852.9, 2.0544
1674.2, 1.0180
```

**Comment by edaphos** [comment]
OK. Calibrating x-axis with X1 = 10 and X2 = 10000, and not using points, I digitized positions along the x-axis, for "0", 10, 100, 1000, 10000 to yield corresponding digitized values of 5.9,  9.9, 71.6, 841.7, 10000.  Must be a "mixed" plot ???

**Comment by edaphos** [comment]
Yes. Unfortunately, this chart (Fig 1) is rather odd. https://www.sciencedirect.com/science/article/pii/S0264410X24001270?via%3Dihub

**Comment by ankitrohatgi** [comment]
Yeah, something is weird with the axis. Distance from 10-100 is not the same as 100-1000

<img width="656" alt="image" src="https://github.com/automeris-io/WebPlotDigitizer/assets/415640/97ac7db8-b9ad-4ca6-93cf-a8e91df802cb">
